contracts/LifMarketMaker.sol

```diff
@@ -162,7 +213,7 @@ contract LifMarketMaker is Ownable {
   }
 
   function() payable {
-    buyLif();
+    getTokens();
   }
 
   function getTokens() payable {
```

Comment:
Changed function name on fallback, and on for getSellPrice on getTokens(), I leave the profit for another PR.